src/api/pubsub.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+const promisify = require('promisify-es6')
+const bs58 = require('bs58')
+var Base64 = require('js-base64').Base64
+var Stream = require('stream')
+var http = require('http')
+
+let activeSubscriptions = []
+
+const subscriptionExists = (subscriptions, topic) => {
+  return subscriptions.indexOf(topic) !== -1
+}
```

Comment:
Can use `subscriptions.includes(topic)` instead of .indexOf(). Perhaps not even needed to wrap in a function?